ariadne/cardano/src/Ariadne/System/FileMode.hs

```diff
@@ -0,0 +1,47 @@
+module Ariadne.System.FileMode
```

Comment:
I think it's better to put this module into `ariadne-core`, because it doesn't depend on anything from Cardano.